sql/OMOP_CDM_sql_server_staging_ddl_indexes.sql

```diff
@@ -0,0 +1,452 @@
+/**************************
+
+Standardized meta-data
+
+***************************/
+
+CREATE TABLE staging_cdm_source
```

Comment:
@whitneygriffith  Instead of duplicating the schema you can do the following
```sql 
SELECT TOP 0 *
INTO dbo.staging_cdm_source
FROM dbo.cdm_source
```
It will create a table `dbo.staging_cdm_source` with the same table schema. It will not create keys, constraints and indexes though.